### PR TITLE
Moving routing status repo to tripkit

### PR DESCRIPTION
**Pull request by thuutin**
### What is new in this Pull Request?

Move routing status repository to tripkit

### CODE REVIEW

**Comment by thuutin** [comment]
[WIP] do not review yet. @thuytrinh

**Comment by thuutin** [comment]
What do you want to do instead? So far I have seen 2 messages shown: `destination equals origin` and `location lies outside of the supported areas`.

**Comment by thuytrinh** [comment]
> What do you want to do instead? So far I have seen 2 messages shown: destination equals origin and location lies outside of the supported areas.

I suggest to undo the part of saving error messages. Then we'll create a new Redmine issue to discuss how to deal with errors with Think, and Jayen. In the mean time, we can just show a generic error (`Unexpected error just occurred` in our translation repo?).

There're some error scenarios on the Routes screen:

* Unable to get GPS locations for case of routing from current locations.
* No Internet connection.
* User errors which server returns (like the 2 cases u mentioned).

**Comment by thuutin** [comment]
Yes. Will do that.

File: TripKitData/build.gradle [hunk 1]
```diff
@@ -13,8 +13,8 @@ android {
   buildToolsVersion versions.buildToolsVersion
 
   defaultConfig {
-    minSdkVersion 10
-    targetSdkVersion 25
+    minSdkVersion versions.proMinSdkVersion
+    targetSdkVersion versions.targetSdkVersion
```
Comment (thuytrinh) [inline TripKitData/build.gradle]: I just helped use those constants in the dependencies file instead. @thuutin

**Comment by thuytrinh** [comment]
Thanks @thuutin 👍

**Pull request merged**